include/boost/beast/core/impl/handler_ptr.ipp

```diff
@@ -52,8 +52,8 @@ handler_ptr(handler_ptr&& other)
 {
     if(other.t_)
     {
-        new(&h_) Handler(std::move(other.handler()));
-        other.handler().~Handler();
+        new(&h_) Handler(std::move(other.h_));
```

Comment:
No `addressof(h)`? Before you didn't have to worry about `T` having overloaded `&`, but now you do.

Reply:
Damn, forgot about that. Done.

Reply:
Also fully-qualify `::new`?

Reply:
Done.